frontend/src/components/ViewActivities/editActivity.js

```diff
@@ -132,23 +133,24 @@ class EditActivity extends React.Component {
    */
   countriesDropdown(ref, tzref, defaultCountry) {
     return (
-      <FormControl as='select' ref={ref} onChange={tzref} defaultValue={defaultCountry}>
+      <Form.Control as='select' ref={ref} onChange={tzref} defaultValue={defaultCountry}>
         {countryList.map((item, index) => {
           return (<option key={index}>{item}</option>);
         })}
-      </FormControl>
+      </Form.Control>
     );
   }
 
 
   render() {
     const activity = this.props.activity;
+    const newAct = this.props.new;
     return (
       <Form className='activity-editor' onSubmit={this.finishEditActivity}>
         {formElements.textElementFormGroup( // TITLE
             'formActivityTitle',          // controlId
             'Title:',                     // formLabel
-            activity[DB.ACTIVITIES_TITLE],// placeHolder 
+            getField(activity, DB.ACTIVITIES_TITLE, 'What will you be doing?'), // placeHolder 
```

Comment:
Constant strings like these could be nice to separate into a grouped constants section, so that later down the line it's very easy to update the text shown to users by just navigating to the constants file/section and updating it there, without needing to find where exactly it's called in the code.

This is not a requirement, but would be nice -- up to you.